### ISSUE TITLE
Multiple UV Layer

### ISSUE THREAD

**Issue by neomonkeus** [niftools/max_nif_plugin#1]
Feature Request Migrated from sf.net by Alex
Needs verification if still applicable, format generally does not support this.

I have another feature on my dream list for NifTools Max plugin: Multiple UV sets support.
I have requested that once and even tried to edit (unsuccessfully) code myself.
For what this might be useful? Here are just some examples:
1. Correct texture seams with simple “patches” applied on top of the base texture often created by excellent artist.
2. Add high resolution details to the main texture e.g. scars, tattoos.. 
3. Add some sort of asymmetry to symmetrical models (e.g. one texture for both legs, so, any texture change results in the identical change of both legs).
4. Interesting effects can be achieved by combination of different maps (base, decal, glow…) that are applied to different UV sets of the same mesh.
5. Using different UV sets it is possible to hide some effects of other maps only on a part of a mesh.
At the present moment this is achieved by creating complex multi layer meshes and other tricks that significantly increase polygons numbers. Official exporters as well as Blender have this feature. I understand that this does not have high priority, I am just reminding, just in case there would be a chance to look at this part of the code.
Thank you.

**Comment by blabbatheorange** [comment]
After some messing around in nifskope, I am able to define multiple UV sets for a skyrim mesh in nifskope, however testing needs to be done if the game reads and applies the data as well as some other settings need to be further explored. If it does work out then the user request is definitely do-able.

**Comment by neomonkeus** [comment]
Yeah I don't think that the nif format supports more than1 uv layer per material. I know we don't support this in Blender Nif Plugin, even though the software itself allows for this.

Testing in-game is required.
